exporter/splunkexporter/metricdata_to_splunk.go

```diff
@@ -0,0 +1,46 @@
+package splunkexporter
+
+import (
+	"fmt"
+	"go.opentelemetry.io/collector/consumer/consumerdata"
+	"go.uber.org/zap"
+)
+
+type splunkMetric struct {
+	Time       int64             `json:"time"`                 // epoch time
+	Host       string            `json:"host"`                 // hostname
+	Source     string            `json:"source,omitempty"`     // optional description of the source of the event; typically the app's name
+	SourceType string            `json:"sourcetype,omitempty"` // optional name of a Splunk parsing configuration; this is usually inferred by Splunk
+	Index      string            `json:"index,omitempty"`      // optional name of the Splunk index to store the event in; not required if the token has a default index set in Splunk
+	Event      string            `json:"event"`                // type of event: this is a metric.
+	Fields     map[string]interface{} `json:"fields"`               // metric data
+}
+
+func metricDataToSplunk(logger *zap.Logger, data consumerdata.MetricsData, config Config) ([]*splunkMetric, int, error) {
+	// this is wrong
+	host := data.Node.ServiceInfo.Name
```

Comment:
If we go with the proposal mapping here https://github.com/tigrannajaryan/rfcs/blob/feature/tigran/log-data-model/text/0097-log-data-model.md#splunk-hec then the host can be fecthed from the "host.hostname" label of `data.Resource`.

Reply:
this has been integrated.